File: Exemplos/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Termica/StatusImpressora.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Termica
{
    class StatusImpressora : TermicaCommand
    {
        readonly int param;

        public StatusImpressora(int param) : base("StatusImpressora")
        {
            this.param = param;
        }

        protected override string FunctionParameters()
        {
            return "\"param\"" + ":" + param;
        }
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_SmartPOS_Ionic-font/src/intentDigitalHubService/TERMICA/Commands/StatusImpressora.ts
import {TermicaCommand} from '../TermicaCommand';

export class StatusImpressora extends TermicaCommand {
  readonly param: number;

  constructor(param: number) {
    super('StatusImpressora');
    this.param = param;
  }

  functionParameters(): object {
    return {param: this.param};
  }
}
